packages/e2e/.env.example

```diff
@@ -0,0 +1,8 @@
+[Required] The initial Mudita OS version on device
+TEST_INITIAL_VERSION=
+
+[Required] Path to Mudita Center binary
+TEST_BINARY_PATH=
+
+[Optional] WebdriverIO log level
+TEST_LOG_LEVEL=
```

Comment:
```suggestion
# [Required] The initial Mudita OS version on device
TEST_INITIAL_VERSION=

# [Required] Path to Mudita Center binary
TEST_BINARY_PATH=

# [Optional] WebdriverIO log level
TEST_LOG_LEVEL=
```

Reply:
and what with `TEST_GITHUB_USER` & `TEST_GITHUB_PASSWORD` or `TEST_GITHUB_TOKEN`.

Reply:
https://github.com/mudita/mudita-center/pull/611/commits/b7e381a9b47bf9ffbf1f014e9fe4df4194fa8563